aea/multiplexer.py

```diff
@@ -263,17 +263,36 @@ async def disconnect(self) -> None:
         async with self._lock:
             if self.connection_status.is_disconnected:
                 self.logger.debug("Multiplexer already disconnected.")
-                await asyncio.wait_for(self._stop(), timeout=60)
                 return
             try:
                 self.connection_status.set(ConnectionStates.disconnecting)
-                await asyncio.wait_for(self._disconnect_all(), timeout=60)
                 await asyncio.wait_for(self._stop(), timeout=60)
                 self.logger.debug("Multiplexer disconnected.")
             except (CancelledError, Exception):
                 self.logger.exception("Exception on disconnect:")
                 raise AEAConnectionError("Failed to disconnect the multiplexer.")
 
+    async def _stop_recv_send_loops(self) -> None:
```

Comment:
Please don't abbreviate `recv` ..it is inconsistent with the code base (spelling things out!) and confusing. Relabel `receive`.

Reply:
done